composables/projectProperties.ts

```diff
@@ -28,10 +32,21 @@ export function useProjectProperties() {
    */
   async function fetchProjectProperties(locale: string) {
     try {
-      return (
-        cachedProjectProperties.value[locale] ||
-        (await $fsxaApi.fetchProjectProperties({ locale }))
-      )
+      const projectPropertiesResponse = await fsxaApi.fetchProjectProperties({
+        locale
+      })
+      let projectProperties = null
+      if (
```

Comment:
We should save the result of the condition in speaking name.
like: 
const isWrappedProjectProperties = <condition>
if (isWrappedProjectProperties) { ... }

This makes it way more easy to understand the code in the future